scss/ckeditor/skins/tao/scss/dialog.scss

```diff
@@ -49,6 +49,10 @@ Comments in this file will give more details about each of the above blocks.
   visibility: visible;
 }
 
+.cke_dialog_container {
+  background: transparent!important;

```

Comment:
can it be done without `!important`?

Reply:
Unfortunately not it's overrides styles with `!important` property provided by dialog plugin